src/foam/core/auth/login/LoginView.js

```diff
@@ -299,9 +299,6 @@ foam.CLASS({
       name: 'signInAction',
       label: 'Sign in',
       buttonStyle: 'PRIMARY',
-      isEnabled: function(data, data$errors_) {
```

Comment:
why remove this?

Reply:
Good question.

Reply:
The issue is just that the values are actually empty until you try to use the form. The browser is suggesting values, but doesn't actually enter them until you click the button, so we think the fields are empty and disable the login button. In this case, we should just do what Anas did and disable the isEnabled for this button only.

Reply:
I pushed just this fix:

commit 8672eac8ca96be608b53d6d2c81c58395b6bf298 (HEAD -> development, origin/development, origin/HEAD)
Author: Kevin Greer <kgrgreer@gmail.com>
Date:   Thu Jul 3 10:16:27 2025 -0300

    Disable isEnabled for LoginView login button because it doesn't work well with browser's password auto-suggest feature.
    PT-FIXES: DOS-61